src/Hosting/Abstractions/src/IStartupFilter.cs

```diff
@@ -6,8 +6,16 @@
 
 namespace Microsoft.AspNetCore.Hosting
 {
+    /// <summary>
+    /// Provides an interface for extending the middleware pipeline with new Configure methods.
```

Comment:
```suggestion
    /// Provides an interface for extending the middleware pipeline with new Configure methods. <see cref="IStartupFilter" /> can be used to add defaults to the beginning or end of the pipeline without having to make the app author explicitly register middleware. 
```

Reply:
Yeah, this will be probably show up weirdly on the docs page (link to the same page you're on). I can modify Pranav's change.